packages/@vuepress/theme-default/components/AlgoliaSearchBox.vue

```diff
@@ -15,8 +16,15 @@
 export default {
   props: ['options'],
 
+  data () {
+    return {
+      placeholder: this.placeholder
```

Comment:
```suggestion
      placeholder: undefined
```

Reply:
`this.placeholder` won't be defined at this moment so you can set the default value of `placeholder` to undefined